.github/CODEOWNERS

```
@@ -84,20 +95,22 @@
 /packages/expo-status-bar                       @brentvatne
 /packages/expo-store-review                     @EvanBacon
 /packages/expo-structured-headers               @douglowder
+/packages/expo-symbols                          @alanjhughes
 /packages/expo-system-ui                        @EvanBacon
 /packages/expo-task-manager                     @tsapeta @bycedric
 /packages/expo-tracking-transparency            @tsapeta
 /packages/expo-updates                          @wschurman @douglowder
 /packages/expo-updates-interface                @wschurman @douglowder
-/packages/expo-video-thumbnails                 @tsapeta
 /packages/expo-video                            @tsapeta @aleqsio @behenate
-/packages/expo-web-browser                      @tsapeta
-/packages/expo                                  @ide @tsapeta @brentvatne
-/packages/jest-expo-puppeteer                   @ide @EvanBacon
+/packages/expo-video-thumbnails                 @tsapeta
+/packages/expo-web-browser                      @alanjhughes
+/packages/expo-yarn-workspaces                  @bycedric
+/packages/htmp-elements                         @EvanBacon
+/packages/install-expo-modules                  @kudo
 /packages/jest-expo                             @brentvatne @EvanBacon
+/packages/jest-expo-puppeteer                   @ide @EvanBacon
+/packages/patch-project                         @kudo
 /packages/pod-install                           @bycedric @EvanBacon
 /packages/uri-scheme                            @bycedric @EvanBacon
+/pakcages/expo-audio                            @alanjhughes @behenate  
```

```suggestion
/packages/expo-audio                            @alanjhughes @behenate  
```
and sort everything again 😅